### PR TITLE
fix(Diagnostics): remove unnneded tenantInfo fetch

### PR DESCRIPTION
**Pull request by artemmufazalov**
After https://github.com/ydb-platform/ydb-embedded-ui/pull/419

### CODE REVIEW

File: src/containers/Tenant/Diagnostics/DetailedOverview/DetailedOverview.scss [hunk 1]
```diff
@@ -4,6 +4,9 @@ $section-title-line-height: 24px;
 .kv-detailed-overview {
     display: flex;
     gap: 20px;
+
+    height: 100%;
```
Comment (artemmufazalov) [inline src/containers/Tenant/Diagnostics/DetailedOverview/DetailedOverview.scss:8]: For correct loader align

File: src/containers/Tenant/Tenant.tsx [hunk 2]
```diff
@@ -127,7 +120,7 @@ function Tenant(props: TenantProps) {
         dispatchSummaryVisibilityAction(PaneVisibilityActionTypes.clear);
     };
 
-    const showBlockingError = tenantStatus === 403 || schemaStatus === 403;
+    const showBlockingError = schemaStatus === 403;
```
Comment (artemmufazalov) [inline src/containers/Tenant/Tenant.tsx:123]: If there is no access, `/viewer/json/describe` should fail with 403 code as well. `/tenantinfo` is too heavy request for that purposes

File: src/containers/Tenant/Diagnostics/Overview/Overview.tsx [hunk 3]
```diff
@@ -1,85 +1,87 @@
...
+        (overviewLoading && !overviewWasLoaded) || (olapStatsLoading && !olapStatsWasLoaded);
 
     const isEntityWithMergedImpl = isEntityWithMergedImplementation(type);
 
-    // There is a circular dependency here. Fetch data depends on children paths
-    // When data in store updated on fetch request,
-    // new object is set there, so source children array is updated
-    // This updates selector, the selector returns a new array, and data is fetched again
-    // To prevent it, shallowEqual, which compares array content, was added
+    // shalloEqual prevents rerenders when new schema data is loaded
```
Comment (artemmufazalov) [inline src/containers/Tenant/Diagnostics/Overview/Overview.tsx:61]: Here still could be data update when preloaded or previously loaded data (`schema` keeps data) is replaced with new data

File: src/containers/Tenant/Diagnostics/Overview/ChangefeedInfo/ChangefeedInfo.tsx [hunk 4]
```diff
@@ -42,27 +41,26 @@ const prepareChangefeedInfo = (
...
     const {error: schemaError} = useTypedSelector((state) => state.schema);
-    const pqGroupData = useTypedSelector((state) => selectSchemaData(state, childrenPaths?.[0]));
 
     if (schemaError) {
         return <div className="error">{schemaError.statusText}</div>;
     }
 
-    if (!data || !pqGroupData) {
+    if (!data || !topic) {
         return <div className="error">No {entityName} data</div>;
```
[the remark this answers is not in the document]
Comment (artemmufazalov) [reply src/containers/Tenant/Diagnostics/Overview/ChangefeedInfo/ChangefeedInfo.tsx:58]: It is without `i18n`, but it seems correct. Could you clarify the problem?

[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/containers/Tenant/Diagnostics/Overview/ChangefeedInfo/ChangefeedInfo.tsx:58]: Should `entityName` be variable?

[the remark this answers is not in the document]
Comment (artemmufazalov) [reply src/containers/Tenant/Diagnostics/Overview/ChangefeedInfo/ChangefeedInfo.tsx:58]: It's a variable. So it should be "No Changefeed Data"

[the remark this answers is not in the document]
Comment (Raubzeug) [reply src/containers/Tenant/Diagnostics/Overview/ChangefeedInfo/ChangefeedInfo.tsx:58]: I don't know what had come to me in the end of previous week) sorry)

**Review by Raubzeug — APPROVED** [review]

**Pull request merged**